Sentry cloudflare

rakyi opened the pull request:
Resolves #4187

ikesau reviewed (commented):
Nice! It's a shame we lose the source map 😢 

I tried visiting https://sentry-cloudflare.owid.pages.dev/deleted/deleted-article but it didn't seem to log the error in Sentry. Did you have to do something else to get it to log?

rakyi commented:
I added a temporary code that raised the error on that URL to test that the integration works, and then I removed it. You could do the same, if you want to double-check.
